File: src/main/java/aristata/programmers/d20190316/ChangeInteger.java
package aristata.programmers.d20190316;

public class ChangeInteger {

    public static void main(String[] args) {
        ChangeInteger changeInteger = new ChangeInteger();

        String testCase1 = "1234";
        System.out.println(changeInteger.solution(testCase1));

        String testCase2 = "-1234";
        System.out.println(changeInteger.solution(testCase2));
    }

    private int solution(String s) {
        return Integer.parseInt(s);
    }
}

File: src/main/kotlin/aristata/programmers/d20190316/ChangeInteger.kt
package aristata.programmers.d20190316

class ChangeInteger {

    fun solution(s: String): Int {
        return Integer.parseInt(s)
    }
}

fun main(args: Array<String>) {
    val changeInteger = ChangeInteger()

    val testCase1 = "1234"
    println(changeInteger.solution(testCase1))

    val testCase2 = "-1234"
    println(changeInteger.solution(testCase2))
}
